### ISSUE TITLE
Missing the `aria-label` attribute from the `BitTypography` component

### ISSUE THREAD

**Issue by Cyrus-Sushiant** [bitfoundation/bitplatform#7988]
To improve accessibility, we need to add the `aria-label` attribute to the `BitTypography` component. This will ensure that screen readers can provide a meaningful description of the component for users with disabilities.
